src/app/containers/MostRead/index.jsx

```diff
@@ -13,6 +13,7 @@ const MostReadContainer = ({
   columnLayout,
   size,
   wrapper,
+  isMostReadPage,
```

Comment:
Could this prop be `serverRenderOnAmp` or something similar, to clearly indicate what this is doing?